packages/millicast-sdk/src/utils/Diagnostics.js

```diff
@@ -28,21 +35,39 @@ const setStats = (stats) => {
   diagnostics.stats.push(stats)
 }
 
-const getDiagnostics = (statsCount = 5) => {
+const initStats = async (statsCount) => {
+  try {
+    webRTCPeer.initStats()
+    await new Promise(resolve => setTimeout(resolve, statsCount * 1010))
+    webRTCPeer.stopStats()
+  } catch (e) {
+    console.error('Could not init stats.')
+  }
+}
+
+const getDiagnostics = async (statsCount = STATS_HISTORY_SIZE) => {
   diagnostics.version = version
   diagnostics.timestamp = Date.now()
   diagnostics.userAgent = window?.navigator?.userAgent || 'No user agent available'
 
-  if (Number.isInteger(statsCount) && statsCount >= 0 && statsCount < STATS_HISTORY_SIZE) {
-    diagnostics.stats = diagnostics.stats.slice(-statsCount)
+  if (!(diagnostics.stats.length > 0)) {
```

Comment:
```suggestion
  if (diagnostics.stats.length) {
```
then switch the blocks below is much better